File: Hyunsuk-Lee/Java/BOJ11022.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class BOJ11022{
  public static void main(String[] args) throws IOException{
    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    int tc = Integer.parseInt(br.readLine());
    for(int i = 1; i < tc+1; i++){
      String[] input = br.readLine().split(" ");
      System.out.println("Case #" + i +": "+ input[0] + " + " + input[1] + " = " + (Integer.parseInt(input[0]) + Integer.parseInt(input[1])));
    }
    br.close();
  }
}

File: Hyunsuk-Lee/Python3/BOJ11022.py
import sys
tc = int(input());

for i in range(tc):
    a, b = sys.stdin.readline().split(" ")
    val = int(a) + int(b)
    line = "Case #"+str(i+1)+":",str(a)+" + "+str(b)+" = "+str(val)
    print(line)
